docs/pages/guides/using-sentry.mdx

```diff
@@ -468,7 +469,52 @@ Using the above instructions, no additional work is needed to integrate Sentry i
 
 ## Usage with EAS Update
 
-In the current state of SDK 50 beta, EAS Update is not yet supported. We're working on making it easier than ever to integrate updates with Sentry, and we plan to release along with SDK 50 proper.
+### Tag Configuration
+Configure Sentry to tag your scope with information about your update. This will allow you to see which errors are happening on which updates in the Sentry dashboard.
+
+Do this in the global scope, as early as possible in your application's lifecycle.
+  
+```js
+import * as Updates from "expo-updates";
+
+Sentry.init({
+  // dsn, release, dist, etc...
+});
+
+Sentry.configureScope((scope) => {
+  scope.setTag("expo-update-group-id", Updates.manifest?.metadata?.updateGroup);
+  scope.setTag("expo-update-id", Updates.updateId);
+  // This will be `true` if the update is the one embedded in the build, and not one downloaded from the updates server.
+  scope.setTag("expo-is-embedded-update", Updates.isEmbeddedLaunch);
+  scope.setTag(
+    "expo-update-debug-url",
+    Updates.isEmbeddedLaunch
+      ? "not applicable for embedded updates"
+      : `https://expo.dev/accounts/${Updates.manifest?.extra?.expoClient?.owner}/projects/${Updates.manifest?.extra?.expoClient?.slug}/updates/${Updates.manifest?.metadata?.updateGroup}`
+  );
+});
+```
```

Comment:
```suggestion
```js
import * as Updates from "expo-updates";

const { metadata, extra } = Updates.manifest;

Sentry.init({
  // dsn, release, dist, etc...
});

Sentry.configureScope((scope) => {
  scope.setTag("expo-update-group-id", metadata?.updateGroup);
  scope.setTag("expo-update-id", Updates.updateId);
  // This will be `true` if the update is the one embedded in the build, and not one downloaded from the updates server.
  scope.setTag("expo-is-embedded-update", Updates.isEmbeddedLaunch);
  scope.setTag(
    "expo-update-debug-url",
    Updates.isEmbeddedLaunch
      ? "not applicable for embedded updates"
      : `https://expo.dev/accounts/${extra?.expoClient?.owner}/projects/${extra?.expoClient?.slug}/updates/${metadata?.updateGroup}`
  );
});
```
```

WDYT about using destructing assignment to shorten the debug URL, for the overall example readability?

Reply:
Looks good, thank you!